src/rail/estimation/algos/lephare.py

```diff
@@ -25,63 +27,51 @@ class LephareInformer(CatInformer):
         err_bands=SHARED_PARAMS,
         ref_band=SHARED_PARAMS,
         redshift_col=SHARED_PARAMS,
-        lephare_config_file=Param(
-            str,
-            "{}/{}".format(os.path.dirname(os.path.abspath(__file__)), "lsst.para"),
-            msg="Path to the lephare config in .para format",
-        ),
-        star_sed=Param(
-            str,
-            "$LEPHAREDIR/sed/STAR/STAR_MOD_ALL.list",
-            msg="Path to text file containing list of star SED templates",
-        ),
-        qso_sed=Param(
-            str,
-            "$LEPHAREDIR/sed/QSO/SALVATO09/AGN_MOD.list",
-            msg="Path to text file containing list of galaxy SED templates",
-        ),
-        gal_sed=Param(
-            str,
-            "$LEPHAREDIR/examples/COSMOS_MOD.list",
-            msg="Path to text file containing list of quasar SED templates",
-        ),
-        star_mag_dict=Param(
+        lephare_config=Param(
             dict,
-            dict(
-                lib_ascii="YES",
+            lp.read_config(
+                "{}/{}".format(os.path.dirname(os.path.abspath(__file__)), "lsst.para")
```

Comment:
I suppose this is fine - but I don't think I've seen the default value being defined as a line of code like this. i.e. lp.read_config(...). The way this was originally seemed more intuitive. With `self.lephare_config = lp.read_config(self.config["lephare_config_file"])` specified in the `__init__` method.